conditional/util/member.py

```diff
@@ -137,4 +137,6 @@ def get_hm(member):
                       HouseMeeting.date).filter(
                           HouseMeeting.date > start_of_year(),
                           MemberHouseMeetingAttendance.uid == member.uid)
+    if only_absent:
+        h_meetings = [hm for hm in h_meetings if hm.attendance_status == "Absent"]
```

Comment:
Are SQLAlchemy queries lazily loaded? If so h_meetings should produce a SQLAlchemy Query object but not actually hit the database until it is used. IF that is the case

```python
if only_absent:
    h_meetings = h_meetings.filter(MemberHouseMeetingAttendance.attendance_status == "Absent")
```

will work nicely and not require evaluating the entire queryset and filtering it in memory

Reply:
Yup it does. Good catch, thanks.